### DISCUSSION TITLE
Can not access http api with 403

### DISCUSSION THREAD

**Discussion by LuoHuacheng** [OneKeyHQ/app-monorepo#4990] [Q&A] [answered]
When I tried to run the code the http error 403 came to me, after some debugging I found this problem happened because of the host but I don't konw how to fix it, can anyone help me ? 
![image](https://github.com/OneKeyHQ/app-monorepo/assets/14214871/5e3d70b5-9111-4184-bfb8-a9a485566120)

**Comment by revan-zhang** [comment, marked as answer]
We have certain restrictions on external calls to the API. Currently, our API has no plans to be open and used by the outside, so direct calls will result in errors.
